app/api/api_v1/routers/admin.py

```diff
@@ -268,10 +277,139 @@ async def request_password_reset(
 
 
 @r.post(
-    "/bulk-imports/cclw/law-policy",
+    "/bulk-imports/cclw/law-policy-dfc",
     response_model=BulkImportValidatedResult,
     status_code=status.HTTP_202_ACCEPTED,
 )
+def import_law_policy_dfc(
+    request: Request,
+    law_policy_csv: UploadFile,
+    background_tasks: BackgroundTasks,
+    db=Depends(get_db),
+    current_user=Depends(get_current_active_superuser),
+    s3_client=Depends(get_s3_client),
+):
+    """Process a Law/Policy documents, families and collections data import"""
+    _LOGGER.info(
+        f"Bulk import request triggered for CCLW Law & Policy data.",
+        extra={
+            "props": {
+                "superuser_email": current_user.email,
+            }
+        },
+    )
+    try:
+        # TODO move into a generator
+        file_contents = law_policy_csv.file.read().decode("utf8")
+        reader = csv.DictReader(StringIO(initial_value=file_contents))
+
+        if reader.fieldnames is None:
+            _LOGGER.info("No fields in CSV!")
+            sys.exit(11)
+        assert validate_csv_columns(reader.fieldnames)
+
+        # TODO validate metadata
+        not_added_import_ids = []
+        existing_import_ids = [id[0] for id in db.query(FamilyDocument.import_id)]
+        _LOGGER.info(
+            "Got existing import ids (CPR Document ID).",
+            extra={"props": {"existing_import_ids_count": len(existing_import_ids)}},
+        )
+
+        pipeline_input_data = InputData(new_documents=[], updated_documents={})
+        family_updates = {}
+
+        for row in reader:
+            row_object = DfcRow(row)
+
+            if row_object.cpr_document_id in existing_import_ids:
+                physical_document_update = physical_document_updated(
+                    row=row_object, db=db
+                )
+                if bool(physical_document_update):
```

Comment:
Assumption that a physical document update will require a pipeline action whereas family update will not?